src/gui/ar/areaTargetScanner.js

```diff
@@ -141,13 +149,31 @@ createNameSpace("realityEditor.gui.ar.areaTargetScanner");
         }
 
         const headerText = 'No scans of this space detected. Make a scan?';
-        const descriptionText = 'This will create a World Object on your edge server.'
+        let randomServerIP = Object.keys(detectedServers).filter(detectedServer => {
+            return !['127.0.0.1', 'localhost', '10.10.10.10', '10.10.10.20'].includes(detectedServer);
+        })[0]; // this is guaranteed to have at least one entry if we get here
```

Comment:
I'm partway through implementing a method for servers to specify that they shouldn't support world objects (like 10.10.10.10 and 10.10.10.20), so we won't need to hard code those when I'm done – but for now this is good!

Reply:
Nice, should be good! I've removed the specific hardcode in this case since it was just something I had for testing in the lab